app/jobs/whats_app_adapter/three_sixty_dialog/create_welcome_message_template_job.rb

```diff
@@ -5,14 +5,41 @@
 module WhatsAppAdapter
   module ThreeSixtyDialog
     class CreateWelcomeMessageTemplateJob < ApplicationJob
-      def perform(organization_id:, template_text:, token:)
+      def perform(organization_id:)
         @organization = Organization.find_by(id: organization_id)
-        @token = token
-        @template_text = template_text
+        @token = WhatsAppAdapter::ThreeSixtyDialog::TokenFetcherService.new.call
```

Comment:
This looks like an obvious class method, doesn't it?

Reply:
Yes, I guess the reason I didn't do it that way is to allow for passing in some params, but I'm not sure we ever will and if we do, we can always update the code.